### PR TITLE
Typography improvements

### PR DESCRIPTION
**Pull request by markvanaalst**
This PR adds improvements to the way rich-text content is displayed.

**The issue:**
Rich-text content coming from CH ONE was parsed from JSON to HTML and rendered using `dangerouslySetInnerHTML`. However this does not apply default typography styles to any of the html elements since the HTML is being "injected" and isn't part of the syntax tree.

**Example:**
image

**Solution:**
This PR introduces two new packages: [chakra-ui-prose](https://chakra-ui.com/community/recipes/prose) and [DOMPurify](https://github.com/cure53/DOMPurify). 

Chakra Prose adds default typography classes to the rich-text html making sure the default margin/padding/font-size etcetera are respected. DOMPurify sanitizes the HTML before it is being rendered to make sure it does not have any HTML that could lead to XSS attacks (I know the risks are low, but it was low-effort to include this).

The implementation is done using the ThemeSwitcher. I am using the extendTheme function to load the theme with the Prose extension. To enable loading our custom font for the fantasy theme I needed to override the default fontFamily setting for paragraphs to let it inherit from the font defined in the Chakra Text component. (by default it uses the fontFamily from the body element)

`extendTheme(
          fantasyTheme,
          withProse({
            baseStyle: {
              p: {
                fontFamily: 'inherit',
              },
            },
          })
        )`

### CODE REVIEW

**Review by thomas-desmond — COMMENTED** [review]
Great catch on how dangerouslySetHtml renders the rich text.

File: src/components/ui/RichTextOutput/RichTextOutput.tsx [hunk 1]
```diff
@@ -51,8 +53,10 @@ export const RichTextOutput: FC<RichTextOutputProps> = ({ content }) => {
   ];
 
   return (
-    <ChakraText>
-      <div dangerouslySetInnerHTML={{ __html: generateHTML(content, extensions) }} />
-    </ChakraText>
+    <Prose>
+      <ChakraText variant={'customFont'} textAlign={'center'}>
```
[the remark this answers is not in the document]
Comment (dylanyoung-dev) [reply src/components/ui/RichTextOutput/RichTextOutput.tsx:57]: I agree with that, I think left align would be better here.  Doesn't look horrible for the questions, but really bad for the outcomes.  Looks at least it's just a property change, which is great!

[the remark this answers is not in the document]
Comment (markvanaalst) [reply src/components/ui/RichTextOutput/RichTextOutput.tsx]: Oops, the textAlign wasn't supposed to be in there. That was an experiment just to see how it looked.

Great catch! ;-)

**Review by dylanyoung-dev — APPROVED** [review]

**Comment by markvanaalst** [comment]
Fixed one minor thing: on the outcome page the headers where not using the custom font.

**Pull request merged**